package.json

```diff
@@ -19,6 +19,7 @@
     "big-integer": "1.6.51",
     "bufferutil": "4.0.5",
     "node-polyfill-webpack-plugin": "1.1.4",
+    "reconnecting-websocket": "^4.4.0",
```

Comment:
I think we should use exact versions. My thought was that any JS dependencies obtained directly via `npm` (in case users wanted to use `npm install` or something) should be exactly the same as those we get through Nix (see #286)